subsys/bluetooth/host/conn_internal.h

```diff
@@ -107,7 +107,7 @@ struct bt_conn_le {
 	bt_addr_le_t init_addr;
 	bt_addr_le_t resp_addr;
 
-	uint16_t interval;
+	uint32_t interval_us;
 	uint16_t interval_min;
 	uint16_t interval_max;
```

Comment:
Should we consider appending the unit of these interval values as well? 
Is it OK that they are not in microseconds when the actual interval is?